examples/values-ez.yaml

```diff
@@ -22,13 +11,18 @@ configs:
         public_key_file: "keys-cert/file_ca_pub.pem"
         private_key_file: "keys-cert/file_ca_key.pem"
         root_cert_file: "keys-cert/fulcio-root.pem"
+
   rekor:
     signer:
       secret:
         name: rekor-private-key
         private_key_file: "keys-cert/rekor_key.pem"
   cosign:
-    BASE_DOMAIN: apps.rosa.p9esn-qgkm3-wkk.o7au.p3.openshiftapps.com
+    BASE_DOMAIN: apps.rosa.vbnm4-edu89-ykc.99ok.p3.openshiftapps.com
+    image:
```

Comment:
Does this need to be here ?

Reply:
the values-ez file is the 1 i use as a convenience - it should be removed from the examples in favor of the other examples/*yaml - I'll remove it - and also, we can now remove all the images specifications from the override values since this PR sets the securesign/internal built images as the defaults. I'll push that change after I test that everything works.

Reply:
removed the values-ez file